### PR TITLE
Add pdv unit to influxdbwriter #5627

### PR DESCRIPTION
**Pull request by froehl**
Add pdv unit to influxdbwriter
refs #5627

### CODE REVIEW

File: lib/perfdata/influxdbwriter.cpp [hunk 1]
```diff
@@ -289,6 +289,7 @@ void InfluxdbWriter::SendPerfdata(const Dictionary::Ptr& tmpl, const Checkable::
 		fields->Set(String("acknowledgement"), FormatInteger(checkable->GetAcknowledgement()));
 		fields->Set(String("latency"), cr->CalculateLatency());
 		fields->Set(String("execution_time"), cr->CalculateExecutionTime());
+		fields->Set(String("unit"), pdv->GetUnit());
```
Comment (dnsmichi) [inline lib/perfdata/influxdbwriter.cpp]: That's not entirely correct here. Above is a loop over all performance data pairs, and this one just uses the last unit value.

**Comment by gunnarbeutner** [comment]
Btw, you can update an existing PR by pushing additional commits to your branch. There's no need to create new PRs.

File: lib/perfdata/influxdbwriter.cpp (hunk 1 above)
Comment (froehl) [reply lib/perfdata/influxdbwriter.cpp]: Ok, thx, i changed it to above loop.

**Comment by dnsmichi** [comment]
I'm leaving this open for discussion. I don't really see the benefit either, but it wouldn't hurt us if the following is added as @spjmurray requested:

- the conditional IsEmpty() check is there
- documentation is added (please do note that this was moved into the features documentation from the config object type chapter in git master)

**Comment by froehl** [comment]
@spjmurray: Yes, we would to use the perf-unit for scripted grafana dashboards.

**Pull request closed without merging**